atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceIntegrationTest.java

```diff
@@ -165,6 +170,25 @@ public void repeatedDropTableDoesNotAccumulateGarbage() {
         assertThat(garbageAfterTest, lessThanOrEqualTo(preExistingGarbageBeforeTest));
     }
 
+    @Test
+    public void testLockTablesStateCleanUp() throws Exception {
+        CassandraKeyValueService ckvs = (CassandraKeyValueService) keyValueService;
+        UniqueSchemaMutationLockTable lockTable = new UniqueSchemaMutationLockTable(
+                new SchemaMutationLockTables(ckvs.clientPool, CassandraTestSuite.cassandraKvsConfig),
+                LockLeader.I_AM_THE_LOCK_LEADER);
+        grabLock(ckvs, lockTable);
+        ckvs.cleanUpSchemaMutationLockTablesState();
+        CqlResult result = CassandraTestTools.readLocksTable(ckvs.clientPool, lockTable);
+        assertEquals(result.getRows().size(), 0);
```

Comment:
Don't use `assertEquals`. Use `hasSize` and `assertThat`. (We really shouldn't be using `assertTrue` either but oh well).